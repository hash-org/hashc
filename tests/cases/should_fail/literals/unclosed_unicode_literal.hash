k := '\u{005a';
