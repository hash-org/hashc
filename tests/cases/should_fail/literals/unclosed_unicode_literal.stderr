error: invalid character escape sequence. expected `}` after a escape sequence
 --> $DIR/unclosed_unicode_literal.hash:1:14
1 |   k := '\u{005a';
  |                ^ here
